examples/hybrid/cli_options.jl

```diff
@@ -105,13 +105,13 @@ function parse_commandline()
         arg_type = Symbol
         default = :none # TODO: change to :zalesak
         "--ode_algo"
-        help = "ODE algorithm [`ARS343`, `IMKG343a`, `ODE.Euler`, `ODE.IMEXEuler`, `ODE.Rosenbrock23` (default), etc.]"
+        help = "ODE algorithm [`ARS343` (default), `IMKG343a`, `ODE.Euler`, `ODE.IMEXEuler`, `ODE.Rosenbrock23`, etc.]"
```

Comment:
Can we do this incrementally and first just change `dt` and `max_newton_iters`, and change the default tomorrow once we see that the first set of changes work? It'll also give us a heads up in terms of how to compare performance.

Reply:
Sure! I have done it in [c36272](https://github.com/CliMA/ClimaAtmos.jl/pull/864/commits/c36272b1bb281551f58849b3fc9ea4e32c36fa76). 
I can trigger a longrun build